fdbclient/include/fdbclient/Tenant.h

```diff
@@ -47,6 +47,15 @@ struct TenantMapEntry {
 	TenantMapEntry();
 	TenantMapEntry(int64_t id, KeyRef subspace);
 
+	Value encode() const { return ObjectWriter::toValue(*this, IncludeVersion(ProtocolVersion::withTenants())); }
```

Comment:
Would it be better to make `encode` also static by passing in TenantMapEntry as an argument.

Reply:
I guess that's a stylistic choice. I think I prefer making it a non-static function on the object rather than a static function that takes the object even though it ends up looking different than `decode`.